src/game/default/g_main.c

```diff
@@ -182,9 +183,16 @@ void G_ResetItems(void) {
 			continue;
 		}
 
+		if (ent->locals.item->type == ITEM_TECH) { // free techs, we'll spawn them again
+			G_FreeEntity(ent);
+			continue;
```

Comment:
Do we want the ability for mappers to place techs specifically, and then we only randomly spawn them when they're requested and the map does *not* include them?